lib/MSSA/MemRegion.cpp

```diff
@@ -571,7 +571,10 @@ ModRefInfo MRGenerator::getModRefInfo(const CallBlockNode* cs) {
     bool ref = hasRefSideEffectOfCallSite(cs);
     bool mod = hasModSideEffectOfCallSite(cs);
 
-    if (mod && ref)
+    /// return ModRef if tracks nothing of an external call
+    if (isExtCall(cs->getCallSite()) && !ref && !mod)
```

Comment:
Can we put the `isExtCall` inside `hasRefSideEffectOfCallSite` or `hasSideEffectOfCallSite` to make the code cleaner? It looks a bit ad-hoc here.